steely/plugins/wednesday.py

```diff
@@ -0,0 +1,11 @@
+"""
+It is Wednesday, my dudes
+"""
+
+__author__ = ('Smurphicus')
+COMMAND = 'wednesday'
+
+def main(bot, author_id, message, thread_id, thread_type, **kwargs):
+    message = bot.fetchThreadMessages(thread_id=thread_id, limit=2)[1]
+    wednesday_message = message + ", my dudes"
```

Comment:
This need to be `message.text` ✌

Reply:
I cant copy paste properly when migraine medicine is doing its thing 😭